cmd/libsnap-confine-private/cgroup-support.c

```diff
@@ -80,6 +80,7 @@ static const char *cgroup_dir = "/sys/fs/cgroup";
 // hybrid or legacy) The algorithm is described in
 // https://systemd.io/CGROUP_DELEGATION/
 bool sc_cgroup_is_v2(void) {
+    char *hide_warning = getenv("SNAPD_HIDE_CGROUPV2_WARNING");
```

Comment:
I would prefer to see this use `getenv_bool` instead

Reply:
ACK, that makes a lot of sense!